src/Storages/StorageReplicatedMergeTree.cpp

```diff
@@ -3540,6 +3606,9 @@ void StorageReplicatedMergeTree::dropPartition(const ASTPtr & query, const ASTPt
 
     String partition_id = getPartitionIDFromQuery(partition, query_context);
 
+    if (query_context.getSettingsRef().insert_quorum)
+        cleanLastPartNode(partition_id);
```

Comment:
Are you sure, that at this moment partition has already dropped and we cannot add a new part to the `/quorum/last_part` after this call to the same partition? And it will be actually dropped after some time (when all replicas will process `DROP_RANGE` entry).

Reply:
Fixed